pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -624,3 +626,19 @@ func getCurrentLibvirtDriverVersion() (string, error) {
 	}
 	return strings.TrimSpace(strings.Split(stdOut, ":")[1]), err
 }
+
+func getCPUFlags() (string, error) {
+	// Check if the cpu flags vmx or svm is present
+	out, err := ioutil.ReadFile("/proc/cpuinfo")
+	if err != nil {
+		logging.Debugf("Failed to read /proc/cpuinfo: %v", err)
+		return "", fmt.Errorf("Failed to read /proc/cpuinfo")
+	}
+	re := regexp.MustCompile(`flags.*:.*`)
+
+	flags := re.FindString(string(out))
```

Comment:
Maybe use this library https://godoc.org/github.com/shirou/gopsutil/cpu#InfoStat ?

Reply:
@guillaumerose this is what I also suggested to @cfergeau but since our usecase is not that much to include this complete library as dependency so I went without it.

Reply:
if we merge https://github.com/code-ready/crc/pull/1396, we will have this library included :angel:

Reply:
@guillaumerose Ah, may be we will merge it soon :)